src/nuked_sc55.cpp

```diff
@@ -107,62 +115,49 @@ const clap_plugin_t* NukedSc55::GetPluginClass()
     return &plugin_class;
 }
 
-// Get the ROM directory from an environment variable if set to a non-empty
-// value. The path must be an absolute directory path
-std::filesystem::path NukedSc55::GetRomEnvDir()
+// Get a list of potential ROM directories from an environment variable 
+// if set to a non-empty value. The paths must be absolute directory paths
+// Entries are separated by the OS PATH separater
+std::vector<std::filesystem::path> NukedSc55::GetRomEnvDirs()
 {
     constexpr char env_rom_dir_name[] = "SOUNDCANVAS_ROM_DIR";
-    std::filesystem::path empty_path = {};
-
-    std::string dir_name = get_env_var(env_rom_dir_name);
-
-    if (dir_name.empty()) {
-        return empty_path;
-    }
-
-    std::filesystem::path env_path(dir_name);
-
-    if (env_path.is_relative()) {
-        return empty_path;
+    std::vector<std::filesystem::path> paths = {};
+    const auto env_dir_list = get_env_var(env_rom_dir_name);
+    if (env_dir_list.empty()) {
+        return paths;
     }
-    
-    std::error_code ec;
-    if (!std::filesystem::is_directory(env_path, ec)) {
-        if (ec) {
-            log("Error getting directory status: %s", ec.message().c_str());
+    for (const auto env_dir : std::views::split(env_dir_list, env_list_sep)) {
+        auto dir = std::filesystem::path(std::string(env_dir.data(), env_dir.size()));
+        if (dir.is_relative()) {
+            continue;
```

Comment:
Maybe log a warning here?

Reply:
Can do so, yeah.

Not that it will help anyone much, the log feature is not usable without changing the path to something that isn't specific to your setup.

Maybe that should be the next feature. A `SOUNDCANVAS_LOG_FILE` env variable maybe?